packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -3631,15 +3631,19 @@ Factors have the following properties:
 The following factor types are supported:
 
 | Factor Type           | Description                                                                                                         |
-| --------------------- | ------------------------------------------------------------------------------------------------------------------  |
+| --------------------- | ------------------------------------------------------------------------------------------------------------------- |
+| `call`                | Software [OTP](http://en.wikipedia.org/wiki/One-time_password) sent via Voice Call to a registered phone number     |
+| `custom`              | A custom [HOTP](https://en.wikipedia.org/wiki/HMAC-based_One-time_Password_algorithm) factor                        |
```

Comment:
@kentonsmith-okta could double-check me just in case, but I believe the Factor Type label should be `token:hotp` here.

Reply:
Fixed.

Reply:
that is correct